components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/ImViewerAgent.java

```diff
@@ -31,6 +31,7 @@
 import ome.model.units.BigResult;
 
 import org.apache.commons.collections.CollectionUtils;
+import org.openmicroscopy.shoola.agents.dataBrowser.DataBrowserAgent;
```

Comment:
It's just so much boilerplate code for a single integer value I'd need. But yes, I should do it properly.